troposphere/static/js/components/images/detail/availability/AvailabilityView.react.js

```diff
@@ -13,35 +13,33 @@ export default React.createClass({
     renderProviderMachine: function(provider_machine) {
         // Hide 'end-dated' provider_machines
         let endDate = provider_machine.get("end_date");
-        if (endDate && endDate.isValid()) {
-            return;
-        }
+        if (endDate && endDate.isValid()) return;
 
         return (
-        <div key={provider_machine.get("id")}>
-            {provider_machine.get("provider").name} -
-            {provider_machine.get("uuid")}
-        </div>
+            <div key={provider_machine.get("id")}>
+                {provider_machine.get("provider").name}
+            </div>
         )
     },
+
     renderBody: function() {
-        var machines = stores.ProviderMachineStore.getMachinesForVersion(this.props.version);
+        let machines = stores.ProviderMachineStore
+            .getMachinesForVersion(this.props.version);
 
-        if (!machines) {
-            return (<div className="content col-md-10">
-                        <div className="loading" />
-                    </div>);
-        }
-        return (
-        <div className="content">
-            {machines.map(this.renderProviderMachine)}
-        </div>
-        );
+        if (!machines) return;
```

Comment:
I prefer:
```
if (!machines) {
    return;
}
```
If you miss that semi-colon, stuff can go awry. We haven't documented this anywhere, but I think we should change it.